content-src/lib/getVideoPreview.js

```diff
@@ -0,0 +1,36 @@
+module.exports = function getVideoPreview(site) {
+  switch (site.provider_url) {
+    case "https://www.youtube.com/":
```

Comment:
I think it might be a better idea to test `site.url` instead with a slightly more forgiving regex, to account for embedly errors/inconsistencies